xkeysnail/input.py

```diff
@@ -140,6 +140,10 @@ def loop(device_matches, device_watch, quiet):
                 if isinstance(waitable, InputDevice):
                     remove_device(devices, waitable)
                     print("Device removed: " + str(device.name))
+                    if interrupt:
+                        if not len(devices):
+                            break
+                        pass
```

Comment:
```suggestion
                    if not device_watch:
                        if not len(devices):
                            break
                        pass
```

Reply:
If we do this, it makes more sense. Then the -i flag can be removed entirely, and exiting becomes the natural behaviour when we have not enabled watching for new device connections.

Reply:
I tested it and this seems to be the best way for everyone, regarding the --watch option.

I use only one control I find it efficient that the xkeysnail is closed when disconnecting and you when staying connected with more than 1 keyboard.

Well, about --watch not identifying my control, maybe this is something to be reviewed, I will test it again.